contracts/stake/tests/stake.rs

```diff
@@ -737,11 +748,15 @@ fn stake_withdraw_unstake() {
 
     // set different block height so that the new notes are easily located and
     // filtered
-    rusk_abi::set_block_height(session, 3);
+    // sets the block height for all subsequent operations to 1
+    let base = session.commit().expect("Committing should succeed");
+    let mut session = rusk_abi::new_session(vm, base, 3)
+        .expect("Instantiating new session should succeed");
+    session.set_point_limit(POINT_LIMIT);
```

Comment:
Is this required? You're setting point_limit 2 lines after too

Reply:
Clearly a copy paste issue. Will fix